### PR TITLE
Fix SCORM manager integration

### PR DESCRIPTION
**Pull request by bfcoder**
Extract package_id from upload
Update scorm manager with data from lms after assignment is created
Bump version to 1.0.11

### CODE REVIEW

File: lib/senkyoshi/canvas_course.rb [hunk 1]
```diff
@@ -161,9 +189,11 @@ def upload_scorm_package(scorm_package, course_id, tmp_name)
           },
           Authorization: "Bearer #{AuthToken.issue_token}",
         ) do |resp|
-          result = JSON.parse(resp.body)["response"]
-          result["points_possible"] = scorm_package.points_possible
-          result
+          result = JSON.parse(resp.body)
+          response = result["response"]
```
Comment (srphillips) [inline lib/senkyoshi/canvas_course.rb:193]: This is more descriptive of the process, but is there any other reason we are making `result` it's own variable if it's never used except to get the response out of it?

Comment (bfcoder) [reply lib/senkyoshi/canvas_course.rb:193]: It is used on line 195.

Comment (srphillips) [reply lib/senkyoshi/canvas_course.rb:193]: You are correct, my apologies!

**Review by srphillips — APPROVED** [review]
Looks good, thanks for fixing this @bfcoder

**Pull request merged**